Cargo.toml

```diff
@@ -54,6 +55,7 @@ instant = { version = "0.1", features = ["wasm-bindgen"] }
 log = "0.4"
 mint = { version = "0.5.6", optional = true }
 once_cell = "1.12"
+cursor-icon = "1.0.0"
```

Comment:
Nitpick: This dependency should be moved below `bitflags` but above `instant` so that the dependencies are alphabetized.